src/styles/blocks/place.scss

```diff
@@ -0,0 +1,49 @@
+.place {
+  &__photos {
+    margin-bottom: 16px;
+
+    @include onDesktop {
+      margin-bottom: 24px;
+    }
+  }
+
+  &__photo-container {
+    position: relative;
+    width: 100%;
+    box-sizing: border-box;
+    max-height: 256px;
+    overflow: hidden;
+    transition: transform $transition-duration;
+
+    &:hover {
+      transform: scale(1.2);
+      z-index: 1;
+    }
```

Comment:
Scaling of pictures is a too big on hover, they overlap each other https://prnt.sc/1tu0ywe